### DISCUSSION TITLE
RPI overheating

### DISCUSSION THREAD

**Discussion by mh60385** [revoxhere/duino-coin#692] [Q&A]
Anyone experiencing overheating on RPI 4 with Duino Mining? 
Now running the PC_Miner.py continuously but its over 80C. 
Other system software on the pi (pihole, mopidy server and duino miner)
Any thoughts?

**Comment by revoxhere** [comment]
It's normal that a Pi without a cooling system will throttle. Mining uses a lot of CPU power, so if you want to run it at high intensity you need a cooler.

**Reply by mh60385** [reply]
good to know thanks for this. is there any way on the rpi to limit cpu use to a lower % so it can still mine and not heat so much ?

**Comment by JamesHFredPH** [comment]
What you will want to do is go into the Resources Folder and Edit the Mining.cfg File.
(For version 2.56 the folder will be "PCMiner_2.56_resources" and the cfg file will be "Miner_config.cfg")

Open the in "Nano", or "Geany"...
(Basically any Text Editor that will not insert unknown control characters.)

Within the Config FIle these 2 Lines will be what you want to change:
          efficiency = 95
          threads = 4
(They are not indented in the cfg file...)

"efficiency" is an Integer that enables a "rough control of Thread CPU Usage".   You would probably need to drop to 20 to avoid throttling.  (Just Guessing though so experiment...)

"threads" is how many threads/processes you want to create for mining.

Dropping that to "2" may stop throttling as well, but I am guess the "efficiency" will have the most affect.

NOTE:  Make of copy of the Original WORKING cfg File just in case your change causes some issue.
